CriteoPublisherSdk/Sources/CR_CdbBid.m

```diff
@@ -95,12 +94,8 @@ - (instancetype)initWithDict:(NSDictionary *)slot receivedAt:(NSDate *)receivedA
     NSNumber *height = slot[@"height"];
     NSString *creative = slot[@"creative"];
     NSString *impId = slot[@"impId"];
-    // Hard coding to 15 minutes for now
-    // TODO: move this default to the config
-    NSTimeInterval ttl = (slot && slot[@"ttl"]) ? [slot[@"ttl"] doubleValue] : defaultValue;
-    if ([cpm doubleValue] > 0 && ttl == 0) {
-      ttl = defaultValue;
-    }
+    NSTimeInterval ttl =
```

Comment:
Same: adding the time unit to the variable name.

Reply:
I doubt there is any value naming the variable that way as `NSTimeInterval` type has [_by definition_](https://developer.apple.com/documentation/foundation/nstimeinterval) a second unit.

Reply:
Ah ok, well it has some value for someone that doesn't know Objective-C such as myself :) But agreed that it is not necessary in this case.